web/modules/custom/usagov_top_navigation_menu/js/addMenuStyles.js

```diff
@@ -23,14 +23,19 @@
           let aElem = listItem.getElementsByTagName("a")[0];
           // console.log(`the list item is: ${aElem}`);
           aElem.setAttribute("href", "#skip-to-h1");
-          if (document.documentElement.lang === "es") {
-            aElem.setAttribute("aria-current", "página");
+          if (document.documentElement.lang == "es") {
+            var p = document.createElement('p');
+            p.innerHTML = 'esta página';
+            p.classList.add('usa-sr-only');
+            var currentPage = listItem.getAttribute("id");
+            console.log(`current-page id: ${listItem.getAttribute("id")}`);
```

Comment:
Do you want this console.log on the dev environment? Or was it just for debugging locally?

Reply:
Nope! Sorry, just removed and pushed it!